build/configs/stm32l4r9ai-disco/pm/defconfig

```diff
@@ -1250,4 +1375,10 @@ CONFIG_IOTBUS_UART=y
 #
 # Security Framework
 #
-# CONFIG_SECURITY_API is not set
+CONFIG_SECURITY_API=y
```

Comment:
Please match the new defconfig with the old one. Like this config was not set in the original defconfig so don't set it in the new config as well.